sage-ios/SAGE/SAGE/PastSemestersViewController.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  PastSemestersViewController.swift
+//  SAGE
+//
+//  Created by Erica Yin on 3/6/16.
+//  Copyright © 2016 Cal Blueprint. All rights reserved.
+//
+
+import Foundation
+
+class PastSemestersViewController: UITableViewController {
+    
+    var currentErrorMessage: ErrorView?
+    var activityIndicator: UIActivityIndicatorView = UIActivityIndicatorView(activityIndicatorStyle: .Gray)
+    var semesters: [Semester]?
+    var previousVC: UIViewController?
+    
+    override func viewWillAppear(animated: Bool) {
+        self.navigationController?.navigationBar.barTintColor = UIColor.lightGrayColor
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.title = "Past Semesters"
+        self.tableView.tableFooterView = UIView()
+        let n: Int! = self.navigationController?.viewControllers.count
+        self.previousVC = self.navigationController!.viewControllers[n-2]
```

Comment:
wherever you initialize this controller, like in the Admin Table View Controller, you can just say "vc.previousVC = self" instead of needing to do this